libflux/flux-core/src/semantic/mod.rs

```diff
@@ -396,12 +396,25 @@ pub struct Analyzer<'env, I: import::Importer> {
     config: AnalyzerConfig,
 }
 
+/// Features used in the flux compiler
+#[derive(Clone, Eq, PartialEq, Debug, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum Feature {
+    /// Enables vectorization of addition expressions
+    VectorizeAddition,
+    /// Enables vectorization
+    VectorizedMap,
```

Comment:
Should this just be named "Vectorize"? Since the rust code is agnostic for where it's vectorized.

Reply:
Maybe, but the actual feature is called `vectorizedMap` so I felt it was better to be consistent with that.